<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="aedDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>19</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/aed.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="aedTextLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AED Plus Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="patientTextLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Patient Diagram</string>
    </property>
   </widget>
   <widget class="QLabel" name="aedAudioLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AED Audio Instructions</string>
    </property>
   </widget>
   <widget class="QFrame" name="aedAudioFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QTextBrowser" name="aedText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>211</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="configLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Patient Config</string>
    </property>
   </widget>
   <widget class="QFrame" name="configFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="bodyTypeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Body Type:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bodyBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Regular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Child</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="pulseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulse:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pulseSetBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="isPulseEvenLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Is Pulse Even:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="pulseEvenBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="isResponsiveLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Is Responsive:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="responsivenessBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="createPatientButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Patient</string>
     </property>
    </widget>
    <widget class="QComboBox" name="isBreathingBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="isResponsiveLabel_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Is Breathing:</string>
     </property>
    </widget>
    <widget class="QLabel" name="isResponsiveLabel_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Condition:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="conditionBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="userActionsLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User Actions</string>
    </property>
   </widget>
   <widget class="QFrame" name="userActionsFrame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="turnOnDeviceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Turn On Device:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PWR ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="responsivenessLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Of Responsiveness:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="responsivenessButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="callHelpLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Call For Help:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="openChestLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Chest:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openChestButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="defibLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach Defib Pads:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="attachDefibButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach</string>
     </property>
    </widget>
    <widget class="QLabel" name="movebackLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Back:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveBackButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
    <widget class="QLabel" name="shockLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Administer Shock:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shockButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Shock!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openAirwaysButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="checkBreathingButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="checkBreathingLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Check For Breathing:</string>
     </property>
    </widget>
    <widget class="QLabel" name="openAirwaysLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Airways:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cprLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>650</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPR</string>
    </property>
   </widget>
   <widget class="QFrame" name="cprFrame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="compressionButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Compression</string>
     </property>
    </widget>
    <widget class="QPushButton" name="breathButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Breath</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="leftElectrode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Place Left Electrode</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rightElectrode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Place Right Electrode</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lcdText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="okState">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="emergencyState">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="airwayState">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="breathState">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>92</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="padState">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="noTouchState">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="cprState">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="patientFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="patientLabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>6</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>patientFrame</zorder>
   <zorder>aedAudioFrame</zorder>
   <zorder>aedDisplay</zorder>
   <zorder>aedTextLabel</zorder>
   <zorder>patientTextLabel</zorder>
   <zorder>aedAudioLabel</zorder>
   <zorder>configLabel</zorder>
   <zorder>configFrame</zorder>
   <zorder>userActionsLabel</zorder>
   <zorder>userActionsFrame</zorder>
   <zorder>cprLabel</zorder>
   <zorder>cprFrame</zorder>
   <zorder>leftElectrode</zorder>
   <zorder>rightElectrode</zorder>
   <zorder>lcdText</zorder>
   <zorder>okState</zorder>
   <zorder>emergencyState</zorder>
   <zorder>airwayState</zorder>
   <zorder>breathState</zorder>
   <zorder>padState</zorder>
   <zorder>noTouchState</zorder>
   <zorder>cprState</zorder>
   <zorder>horizontalFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
